benchmarks/similarity_scores_time_benchmark.py

```diff
@@ -0,0 +1,147 @@
+"""
+Benchmark time consumption and scores for K-means and most frequent strategies.
+
+We use the traffic_violations dataset to benchmark the different dimensionality
+reduction strategies used in similarity encoding.
+
+Parameters that are modified:
+- Number of rows in datasets: 10k, 20k, 50k, 100k and nuniques.
+- Hashing dimensions: 2 ** 14, 2 ** 16, 2 ** 18, 2 ** 20
+- Ngram-range: (3, 3), (2, 4)
+"""
+
+import warnings
+from time import time
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from sklearn import pipeline, linear_model, model_selection
+from sklearn.compose import ColumnTransformer
+from sklearn.preprocessing import OneHotEncoder
+
+from dirty_cat import SimilarityEncoder
+from dirty_cat.datasets import fetch_traffic_violations
+
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
Which FutureWarning is showing up?

Reply:
This one: "Default solver will be changed to 'lbfgs' in 0.22. Specify a solver to silence this warning."

Reply:
OK, how about we suppress it by using "linear_model.LogisticRegression(solver='lbfgs')"?

Reply:
When I do that it asks to increase the number of iterations (it fails to converge), how much should I put?